platform/linux-generic/odp_thread.c

```diff
@@ -88,7 +88,14 @@ int _odp_thread_init_global(void)
 
 int _odp_thread_term_global(void)
 {
-	int ret;
+	int ret, num;
+
+	odp_spinlock_lock(&thread_globals->lock);
+	num = thread_globals->num;
+	odp_spinlock_unlock(&thread_globals->lock);
+
+	if (num)
+		ODP_DBG("%u threads have not called odp_term_local().\n", num);
```

Comment:
This could be ODP_ERR() since ODP_DBG prints out only with a debug build. I think we'd like to see this by default on all application logs if they violate the proper termination sequence.

Reply:
I was just about to ask about this. Yes, ODP_ERR is better here.